mobile/src/main/java/org/openhab/habdroid/util/ExtensionFuncs.kt

```diff
@@ -455,13 +456,18 @@ fun ServiceInfo.addToPrefs(context: Context) {
     val port = port.toString()
     Log.d(Util.TAG, "Service resolved: $address port: $port")
 
+    val wifiManager = context.applicationContext.getSystemService(Context.WIFI_SERVICE) as WifiManager
+    val wifiInfo = wifiManager.connectionInfo
+    val wifiSsid = if (wifiInfo.networkId == -1) null else wifiInfo.ssid.removeSurrounding("\"")
```

Comment:
This seems kind of magical. I think I'd rather leave the SSID empty for automatically created servers.

Reply:
If the user manually configures a second server and at some point the app switches to the first server magically, a snackbar is shown to inform the user. Then the user knows about this feature or he has seen it before when adding the second server.

Reply:
Still, I don't think second-guessing the user's intent is a good idea.

Reply:
Maybe a dialog or snackbar asking the user whether he wants to bind the found server to the current WiFi network would be a decent compromise?

Reply:
Guess, since it is important a Dialog would be sufficient 👍

Reply:
I'd rather not add the wifi ssid at all instead of showing a dialog. This is probably too special for most users.